integration-tests/tests/src/tests/serialization.ts

```diff
@@ -312,57 +213,18 @@ const testSetups: TestSetup[] = [
         const p1 = realm.create(PlaylistWithId, {
           _id: 1,
           title: "Playlist 1",
-          songs: [
-            s1,
-            s2,
-            s3,
-            {
-              _id: 4,
-              artist: "Unique artist 1",
-              title: "Unique title 1",
-            },
-            {
-              _id: 5,
-              artist: "Unique artist 2",
-              title: "Unique title 2",
-            },
-          ],
+          songs: [s1, s2, s3],
         });
         const p2 = realm.create(PlaylistWithId, {
           _id: 2,
           title: "Playlist 2",
-          songs: [
-            {
-              _id: 6,
-              artist: "Unique artist 3",
-              title: "Unique title 3",
-            },
-            {
-              _id: 7,
-              artist: "Unique artist 4",
-              title: "Unique title 4",
-            },
-            s3,
-          ],
+          songs: [s3],
           related: [p1],
         });
         const p3 = realm.create(PlaylistWithId, {
```

Comment:
again, not sure if all these objects for testing are valuable as they are not meaningfully different now

Reply:
I'd be okay with you removing those, as they're testing a detail of the implementation that we're no longer doing.